LINK : warning LNK4075: ignoring '/INCREMENTAL' due to '/OPT:ICF' specification

I get a large number of these warnings when building c++ x64/Release on Windows 11.

```
"C:\Users\bernard\source\repos\ice\cpp\msbuild\ice.proj" (default target) (1) ->
       "C:\Users\bernard\source\repos\ice\cpp\msbuild\ice.sln" (default target) (4) ->
       "C:\Users\bernard\source\repos\ice\cpp\src\ice2slice\msbuild\ice2slice.vcxproj" (default target) (38) ->
       (Link target) ->
         LINK : warning LNK4075: ignoring '/INCREMENTAL' due to '/OPT:ICF' specification [C:\Users\bernard\source\repos
       \ice\cpp\src\ice2slice\msbuild\ice2slice.vcxproj]
```

--- a/cpp/msbuild/ice.cpp.props
+++ b/cpp/msbuild/ice.cpp.props
@@ -16,7 +16,6 @@
   <PropertyGroup>
     <IceSrcRootDir>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)\..'))</IceSrcRootDir>
     <IntDir>$(Platform)\$(Configuration)\</IntDir>
-    <LinkIncremental>true</LinkIncremental>
   </PropertyGroup>
 
   <Choose>
@@ -74,6 +73,9 @@
     <ResourceCompile>
       <PreprocessorDefinitions>_DEBUG</PreprocessorDefinitions>
     </ResourceCompile>
+    <Link>
+      <LinkIncremental>true</LinkIncremental>
+    </Link>
   </ItemDefinitionGroup>
 
   <!-- Common properties for all release builds and platforms. -->
@@ -86,6 +88,7 @@
     <Link>
       <OptimizeReferences>true</OptimizeReferences>
       <EnableCOMDATFolding>true</EnableCOMDATFolding>
+      <LinkIncremental>false</LinkIncremental>
     </Link>
   </ItemDefinitionGroup>
 

